### DISCUSSION TITLE
Frequency - number of decimal places

### DISCUSSION THREAD

**Discussion by undershot1** [fhdm-dev/scoppy#108] [General]
Hi, I'm using Scoppy to tune some VCOs, and need to measure exactly 1046.5Hz, but as soon as the frequency goes above 1000Hz, it naturally and understandably changes to kHz. Is there a way to increase the granularity so the frequency maybe shows 1.###kHz? I know it's not quite the 4 decimal places I need, but 3 would more or less get me there.

Also, I'm the creator of the wavr.info frontend PCB. Love the simplicity of the schematics and the software you've produced!

**Comment by fhdm-dev** [comment]
Hi,

Unfortunately there's nothing you can do in the current version of the app to get exactly what you want. However, the 'Snapshot measurement' screen does show 3 decimal places. You just need to close it and open it again to refresh the values (I know, that's very annoying and might not be practical for what you are doing). And in case you didn't know, the quickest way to open the measurement snapshot window is to tap and hold where the on-screen measurements are displayed.

I definitely agree that the frequency should show more significant digits, I'm just not sure how many should be shown in the the 'on-screen' measurements. I don't want to clutter up the display and I also need to leave room for extra channels (hopefully it won't be too long before the app has math channels). I need to think about this some more.

And I think your PCB is great. Using 3.5mm jacks is a clever way to keep the costs down :+1: 

:smile:
